crates/cxx-qt-lib/include/common.h

```diff
@@ -33,6 +33,13 @@ drop(T& value)
 template<typename T>
 QString
 toQString(const T& value)
```

Comment:
This template should no longer be necessary.
If `toString()` is already a member function, we don't need to create a free function `toQString`, but can just access it directly via CXX:
i.e.:
```rust
// Maybe with #[doc(hidden)], depending on the situation
#[rust_name="to_qstring"]
fn toQString(self: &T);
```

Please remove this function and replace the existing references with direct member access.

Reply:
The idea of this template is to allow types to have more flexibility in how they define `to_qstring()`. Some types have extra parameters for `toString`. For others, we might not want to have a `to_qstring` method at all (rather than relying on `#[doc(hidden)]`. Or we might decide not to have methods like that at all and use `From<&Type> for QString`, which is more idiomatic. So by using this template, types aren't "locked in" to a particular implementation; we can keep the `Display` logic separate from everything else.

That said, I of course defer to you. What should be the rubric for whether or not to use `#[doc(hidden)]`?